src/tests/test_utils/common_test_utils/include/common_test_utils/postgres_helpers.hpp

```diff
@@ -3,6 +3,7 @@
 //
 
 #pragma once
+
 #include <gtest/gtest.h>
 #include <stdlib.h>
```

Comment:
and... Namespace!

Reply:
This file appeared after I created this PR. May I do it after this PR?